### DISCUSSION TITLE
大腦活動降低 研究：手機成癮殘害創造力

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#1089] [General]
智慧手機時代增加便利性，但許多需要時間歷練的作品，如歌曲、電影、小說等創作都失去光芒。

先前研究就發現手機成癮症會干擾大腦認知功能，並降低創造性

科學家為了證明此論點還做了一項實驗

實驗結果發現:，智慧手機成癮者流暢性、靈活性和獨創性方面得分較低。成癮者大腦前額葉皮層和顳葉區域並不活躍。

之前就有聽說很多優秀歌手都會早起給自己一段沒有手機的時間，因為那是他們最有靈感的時候
我覺得我應該也算手機成癮者，每次洗完澡很長都要開手機檢查有沒有人傳訊息給我(大多時候根本沒訊息，然後就莫名其妙開始滑起ig了)，讀書時也很常會因為無意識的滑起手機浪費了好多時間也降低了專注力
真的要來好好戒手機了...大家一起努力! 擺脫手機的束縛!
https://udn.com/news/story/6897/6798134

**Comment by peggy0329** [comment]
手機內容一次出現太多雜訊真的會讓人腦袋有不適感，就像文章所說的高度專注的分心狀態。
我這週的手機使用報告顯示我平均每天使用手機6小時...
突然覺得我太過頻繁關注社群內容
有時候滑到都沒東西還是會想繼續滑，有一種安全感？哈哈哈
但為了不降低專注力，應該要練習試著放下手機了

**Comment by ru0372572** [comment]
連假或寒暑假放太久的時候就會覺得手機玩到膩了哈哈，覺得自己不是手機成癮者，因為事情一多還是可以放下手機專心做事，也許所謂分心只是逃避的其中一種方法?不過創造力降低我是滿同意的，畢竟什麼都可以查網路，各種創意讓自己的反而被限縮起來。

**Comment by s96015welson** [comment]
我自己也有點手機成癮了，感覺生活離不開手機，沒事也想滑一下，因此生活中零碎的時間都在滑手機。真的該好好重視這個問題了，之前無意發現爬山事個滿好的戒手機方式，因為收不太到訊號，而且爬完很累就不會想去用手機了，比靜態活動效果好。

**Comment by a45320614** [comment]
我認為比起對手機上社群媒體的成癮，我個人更像是對資訊的成癮，透過簡單的手指動作，瞬間就能獲取來自世界各地的大量資訊，製造有趣，悲傷，快樂，憤怒等等會刺激大腦的各種情緒，漸而對資訊感到上癮，過著這樣日復一日被動接收訊息的生活，最後也只會變成一個沒有思考能力及創造能力的軀殼。

**Comment by Jzd314** [comment]
現在人手一支手機，查詢資訊方便太多，以前遇到問題是找同伴交流一起想，現在是手機拿起來就可以直接查到，與其說思考能力下降，不如說思考能力沒有提升

**Comment by BlahFish1006** [comment]
「高度專注的分心狀態」這個形容讓我覺得非常可怕， 因為這就隱含了大腦會將專心的狀態，
逐漸框架於在滑手機時的大腦活動程度，這表示思考的深度將不斷退化。
網路中的訊息流通實在太快速，尤其是社群軟體和短影音，現代人往往無意識間傾向用更短的時間來吸收更多的資訊，
但這種資訊的接收程度，我認為常常僅介在「有接收與沒接收之間」，
沒有經過思考的、被動的接收知識，就真的等於學習到了嗎？那可不一定。

**Comment by wenchi9023** [comment]
還記得考大學的時候我把手機上的社群媒體都戒掉了，專注力有明顯上升，也比較不會緊張和焦慮
在這資訊爆炸、訊息流通迅速的時代，如何在吸取新知以及保有獨立批判能力中取得平衡真的是現代人都需要去省思的問題
